cache/sharded_cache.cc

```diff
@@ -230,4 +227,26 @@ int ShardedCache::GetNumShardBits() const { return BitsSetToOne(shard_mask_); }
 
 uint32_t ShardedCache::GetNumShards() const { return shard_mask_ + 1; }
 
+inline hash_t ShardedCache::HashSlice(const Slice& s) const {
+  uint64_t in_high64 = static_cast<uint64_t>(*s.data());
+  uint64_t in_low64 = static_cast<uint64_t>(*(s.data() + 8));
+  uint64_t out_high64, out_low64;
+  BijectiveHash2x64(in_high64, in_low64, &out_high64, &out_low64);
+  hash_t hash;
+
+  // hash[0] = out_high64 >> 32;
```

Comment:
If I use the hashes that are commented, I get collisions. The hashes below are the original ones, used before this PR.